bindings/py/tests/algorithms/temporal_memory_test.py

```diff
@@ -38,15 +38,51 @@ def testCompute(self):
     self.assertTrue( active.getSum() > 0 )
 
 
-  @pytest.mark.skipif(sys.version_info < (3, 6), reason="Fails for python2 with segmentation fault")
```

Comment:
isn't this still valid? (we don't test on py2, but someone still may run the repo on python2)